actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/OrgBulkUploadActor.java

```diff
@@ -42,6 +42,8 @@
     JsonKey.HASHTAGID,
   };
 
+  private int pos = 0;
```

Comment:
Remove pos instance variable. Multiple OrgBulkUploadActor instances can modify this variable at the same time.